src/main/java/com/rpg2014/wrappers/MinecraftDynamoWrapper.java

```diff
@@ -141,8 +127,9 @@ private void setItem(HashMap<String, AttributeValue> itemKey, HashMap<String, At
         }
```

Comment:
In the above catch block, we should rethrow some sort of error, otherwise the client will thing that this operation succeeded.  In this case I think it should be an InternalServerError, cause something went wrong with our server.  Also I think you should move the DynamoDbError catch into here as well.